### PR TITLE
release 3.5.1 to users

### PR DESCRIPTION
**Pull request by mgallien**
Signed-off-by: Matthieu Gallien <matthieu.gallien@nextcloud.com>

### CODE REVIEW

**Comment by mgallien** [comment]
I tested and the new extra mac fields are properly sent !!!!

**Review by claucambra — APPROVED** [review]

File: config/config.php [hunk 1]
```diff
@@ -83,6 +83,9 @@
 				'versionstring' => $stableVersionString,
 				'downloadurl' => $stable_mac_url . 'Nextcloud-' . $stableVersion . '.pkg',
 				'web' => 'https://nextcloud.com/install/?pk_campaign=clientupdate#install-clients',
```
Comment (tobiasKaminsky) [inline config/config.php:85]: Not directly related, but with our new webpage I think there is no need of "pk_campaign=clientupdate#install-clients", but the pure https://nextcloud.com/install is enough.

Might be worth changing it?

Comment (tobiasKaminsky) [reply config/config.php:85]: Let me do it in another PR…🙈

Comment (mgallien) [reply config/config.php:85]: sure
I was too focused on the task to realize

**Pull request merged**